device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/ReconnectionNotifier.java

```diff
@@ -1,32 +0,0 @@
-/*
- *  Copyright (c) Microsoft. All rights reserved.
- *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
- */
-
-package com.microsoft.azure.sdk.iot.device.transport;
-
-import com.microsoft.azure.sdk.iot.device.exceptions.TransportException;
-
-public final class ReconnectionNotifier
```

Comment:
The purpose of this class was to spawn a thread to notify the transport layer when a connection was lost and to handle the reconnection logic. Now that we have a dedicated thread for this, we can delete this class